dev/conductor/core/lib/src/globals.dart

```diff
@@ -81,6 +82,18 @@ String? getValueFromEnvOrArgs(
     'to be provided!');
 }
 
+bool getBoolFromEnvOrArgs(
```

Comment:
Should we add some tests for this function?

Reply:
done